pkg/ocm/clusters.go

```diff
@@ -740,6 +739,24 @@ func (c *Client) DeleteCluster(clusterKey string, bestEffort bool,
 	return cluster, nil
 }
 
```

Comment:
@stevekuznetsov We will need some unit test coverage on these new functions for our CI to pass. We don't yet have a concrete pattern for unit testing when the OCM SDK is being used, but there are reasonable examples of it being done in `pkg/ocm/kubeletconfig_test.go` 

cheers,
Rob